src/consts/mpitrampoline.jl

```diff
@@ -201,8 +201,8 @@ const MPI_Win_errhandler_fn = MPI_Win_errhandler_function
 
 @const_ref MPI_ARGV_NULL Ptr{Cchar} unsafe_load(cglobal((:MPICONSTANTS_ARGV_NULL, libmpiconstants), Ptr{Ptr{Cchar}}))
 @const_ref MPI_ARGVS_NULL Ptr{Cchar} unsafe_load(cglobal((:MPICONSTANTS_ARGVS_NULL, libmpiconstants), Ptr{Ptr{Ptr{Cchar}}}))
-# MPI_UNWEIGHTED Ptr{Cvoid} unsafe_load(cglobal((:MPICONSTANTS_UNWEIGHTED, libmpiconstants), Ptr{Cint}))
-# MPI_WEIGHTS_EMPTY Ptr{Cvoid} unsafe_load(cglobal((:MPICONSTANTS_WEIGHTS_EMPTY, libmpiconstants), Ptr{Cint}))
+@const_ref MPI_UNWEIGHTED Ptr{Cvoid} unsafe_load(cglobal((:MPICONSTANTS_UNWEIGHTED, libmpiconstants), Ptr{Cint}))
+@const_ref MPI_WEIGHTS_EMPTY Ptr{Cvoid} unsafe_load(cglobal((:MPICONSTANTS_WEIGHTS_EMPTY, libmpiconstants), Ptr{Cint}))
```

Comment:
@eschnett the tests appear to be failing with MPItrampoline. How should these be defined?

Reply:
libload_time_mpi_constants.so seems to be not correctly generated. All integer pointer constants are missing.